helm/idol-library/templates/_ingress.tpl

```diff
@@ -50,9 +61,24 @@ spec:
 {{- end }}
   rules:
   - http:
+      paths:
+{{- range $portType, $pathKey := $portmapping -}}
+{{- if $pathKey }}
```

Comment:
metrics/prometheus should be handled by `$paths` pass in mechanism, not by default handling.

Reply:
They're handled by the `$portmapping` variable, they're not in the default handling.

Reply:
Sorry, had out of date view.